browser-test/login.test.js

```diff
@@ -0,0 +1,76 @@
+import {Builder, By, until} from "selenium-webdriver";
+import {
+  getDriver,
+  getElementByXPath,
+  urls,
+  initialData,
+  executeCommand,
+} from "./utils";
+
+const firefox = require("selenium-webdriver/firefox");
+
+describe("Selenium tests for <Login />", () => {
+  let driver;
+  beforeAll(async () => {
+    jest.setTimeout(10000);
+    await executeCommand("./browser-test/clear_data.py", (message) => {
+      console.log(message);
+    });
+    await executeCommand("./browser-test/initialize_data.py", (message) => {
+      console.log(message);
+    });
```

Comment:
let's avoid duplicating code like:

```javascript
await executeCommand("./browser-test/initialize_data.py", (message) => {
  console.log(message);
});

```

In every test file, please create 2 reusable test functions: `initializeData` and `clearData`.